build-config

```diff
@@ -9,12 +9,12 @@ ROOT=$(PWD)
 INCLUDE_DIR=$(ROOT)/include
 
 INC_DIRS=-I$(INCLUDE_DIR)/mme-app -I$(INCLUDE_DIR)/s11 -I$(INCLUDE_DIR)/s1ap \
-	-I$(INCLUDE_DIR)/s6a \
+	-I$(INCLUDE_DIR)/s6a -I$(ROOT)/../freediameter/build/include -I$(ROOT)/../freediameter/include\
 	-I$(INCLUDE_DIR)/common
 
 HSS_INC_DIRS = -I$(INCLUDE_DIR)/common -I$(INCLUDE_DIR)/hss
 
-CC := gcc
+CC := gcc -L/home/cord/bad/mme/freediameter/build/libfdproto -L/home/cord/bad/mme/freediameter/build/libfdcore
```

Comment:
Avoid paths to local directories.
Freediameter from system is used. Include and lib paths are not needed for FD. 
In case local version is needed then we need to define relative paths, so in absence of local FD system installed FD will be used.

Reply:
Have updated to keep the build-config as it was